osu.Game/Overlays/FirstRunSetup/ScreenImportFromStable.cs

```diff
@@ -122,13 +122,13 @@ private void updateStablePath()
             stableLocatorTextBox.Current.Value = storage.GetFullPath(string.Empty);
             importButton.Enabled.Value = true;
 
-            bool available = legacyImportManager.CheckHardLinkAvailability();
-            Logger.Log($"Hard link support is {available}");
+            bool available = legacyImportManager.CheckSongsFolderHardLinkAvailability();
+            Logger.Log($"Hard link support for beatmaps is {available}");
 
             if (available)
             {
                 copyInformation.Text =
-                    "Data migration will use \"hard links\". No extra disk space will be used, and you can delete either data folder at any point without affecting the other installation. ";
+                    "Beatmap migration will use \"hard links\". No extra disk space will be used, and you can delete beatmaps from either installation at any point without affecting the other. ";
```

Comment:
Just leave these messages. There's no need to mention beatmaps. For the 99% of users this applies to all files, not just beatmaps.